src/types.ts

```diff
@@ -27,8 +27,14 @@ export interface LocaleOptions {
   locale?: Locale
 }
 
+export interface FirstWeekContainsDate {
+  firstWeekContainsDate?: FirstWeekContainsDay
```

Comment:
As discussed in the chat:

```suggestion
export interface FirstWeekContainsDateOptions {
  firstWeekContainsDate?: FirstWeekContainsDate
```